app/code/Magento/Analytics/Model/Config/Backend/CollectionTime.php

```diff
@@ -55,9 +55,9 @@ public function __construct(
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      *
-     * {@inheritdoc}. Set schedule setting for cron.
+     * @inheritdoc. Set schedule setting for cron.
```

Comment:
Incorrect change.

Reply:
Hello @orlangur 
I have updated code can you review it.